bitnami/kafka/templates/broker/statefulset.yaml

```diff
@@ -8,7 +8,7 @@ SPDX-License-Identifier: APACHE-2.0
 apiVersion: {{ include "common.capabilities.statefulset.apiVersion" . }}
 kind: StatefulSet
```

Comment:
I see references to broker.autoscaling.enabled where it should be broker.autoscaling.hpa.enabled